scripts/docker/dockerfile_fake

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:bionic
+RUN apt-get update && apt-get install -y sudo && rm -rf /var/lib/apt/lists/*
+RUN sudo useradd -m -s /bin/bash -G sudo serac
+
+WORKDIR "/home/serac"
+USER serac
```

Comment:
What is the purpose of this file? left over trial-and-error?

Reply:
Yeah, I thought it might be useful to check it in for future testing purposes

Reply:
Comment at the top then